src/frontend/Dockerfile

```diff
@@ -22,9 +22,12 @@ FROM frontend-deps AS drive-dev
 
 WORKDIR /home/frontend/apps/drive
 
+# Add local node_modules/.bin to PATH
+ENV PATH="/home/frontend/apps/drive/node_modules/.bin:$PATH"
```

Comment:
As the working directory is in the directory containing the node_modules, I don't understand why this modification is needed.
What error do you when running `make build-frontend-dev` ?